token/rust/src/token.rs

```diff
@@ -172,6 +180,7 @@ where
     #[allow(clippy::too_many_arguments)]
     pub async fn create_mint<'a, S2: Signer>(
         client: Arc<dyn ProgramClient<T>>,
+        program_id: &Pubkey,
```

Comment:
```suggestion
        program_id: &'a Pubkey,
```
Oops, sorry I missed this